react/components/ProductAssemblyOptions/ProductAssemblyOptionItemCustomize.tsx

```diff
@@ -133,7 +133,7 @@ const ProductAssemblyOptionItemCustomize: FC<Props> = ({
         {...buttonProps}
       >
         <div
-          className={`${handles.productAssemblyOptionItemCustomize__label} c-action-primary t-action`}
+          className={`${handles.productAssemblyOptionItemCustomizeLabel} c-action-primary t-action`}
```

Comment:
Remember to update this `productAssemblyOptionItemCustomize__label` css handle at README.

Reply:
@igorbrasileiro its done ;)